ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml

```diff
@@ -4,7 +4,7 @@ chain:
   - as: dump-management-cluster
     cli: latest
     commands: |-
-      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)-mgmt"
+      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
```

Comment:
@sjenning the issue is the reverse actually and introduced in d104063c6855356363f4b58e023b51b667f48f11: The create steps need to have the `-mgmt` suffix.

Reply:
https://github.com/openshift/release/pull/31713

Reply:
IIRC we can't use the same names for the mgmt cluster and the hostedcluster in it, that is why that suffix was added in the first place